masschallenge/impact-api
# AC-4812: Switch to using django-accelerator custom user model

Work done:
- Got rid of short_name and full_name
- Mostly switch to email rather than username.  The oauth authenticator still wants to use username, so it gets used there.  This likely paves the way to getting rid of username altogether, but that's for another ticket.
- Use the EmailModelBackend from django-accelerator

To test:
- `make build`
- `make load-remote-db DB_FILE_NAME=AC-4812.sql.gz`
- Run tests
- Make sure you can login in all the ways you expect and can run all the API calls you expect to be able to run both from the browsable API and Postman.

Note: Coverage report expected to say:
```
Name                   Stmts   Miss  Cover
------------------------------------------
simpleuser/apps.py         3      3     0%
simpleuser/models.py      32     32     0%
------------------------------------------
TOTAL                    390     35    91%
```
since these files have been removed from the repo and then brought back in through django-accelerator.
